config/application.yml.default

```diff
@@ -60,6 +60,9 @@ broken_personal_key_window_start: '2021-07-29T00:00:00Z'
 broken_personal_key_window_finish: '2021-09-22T00:00:00Z'
 component_previews_enabled: false
 component_previews_embed_frame_ancestors: '[]'
+compromised_password_randomizer_value: 1000
+compromised_password_randomizer_threshold: 900
```

Comment:
Maybe we should override these in local development to force it to be checked every time, to simplify testing?

Reply:
that works a good idea.